go.mod

```diff
@@ -101,6 +101,7 @@ require (
 	github.com/google/pprof v0.0.0-20201117184057-ae444373da19
 	github.com/gordonklaus/ineffassign v0.0.0-20210103220932-664217a59c00
 	github.com/gorilla/mux v1.7.4
+	github.com/gosnmp/gosnmp v1.29.0
```

Comment:
Note for reviewers: Adding `github.com/gosnmp/gosnmp` dep and keeping `github.com/soniah/gosnmp` dep for now.

`github.com/soniah/gosnmp` is currently used by snmp_listener and trap. I will migrate then to use `github.com/gosnmp/gosnmp` in a separate PR.